src/store/page/page-element.ts

```diff
@@ -72,17 +65,40 @@ export class PageElement {
 	 * @return A new page element object containing the loaded data.
 	 */
 	public static fromJsonObject(
-		json: JsonObject,
+		json: JsonObject | undefined,
```

Comment:
you are correct. i changed it because i encountered a bug once where json was null. i think this happens when no root element is defined in a page. this shouldn't be handeled here.

i'm going to revert it

Reply:
done ✅